lib/src/user.dart

```diff
@@ -54,8 +54,10 @@ class User {
         'user_metadata': userMetadata,
         'aud': aud,
         'email': email,
+        'phone': phone,
         'created_at': createdAt,
-        'confirmed_at': emailConfirmedAt,
```

Comment:
Since `confirmed_at` is also still used, it might be better to restore it here!

Reply:
yeah, keep it. It was deprecated, but not removed

Reply:
Alright! I'll get it added back. Should it point to the same email verified value?

Reply:
@HarryET Now that you mention it, I do think it should be pointing to `confirmedAt` instead of `emailConfirmedAt` for better readability. Great catch! 

I think you will need to add `// ignore: deprecated_member_use_from_same_package` like how it's done at line 44 to avoid analysis warning.